frontend/components/ViewEntry.test.js

```diff
@@ -104,7 +110,49 @@ describe('file is displayed', () => {
     fetchMock.post('/change-metadata/dummytoken', { throws : new Error("Network fail") });
     await wrapper.find('.input-group-btn button').trigger('click');
     await Vue.nextTick(); // wait for request
-    expect(wrapper.vm.file.name).not.toBe('new name');
+    expect(wrapper.vm.file.name).not.toBe('new name.odt');
     expect(wrapper.vm.error).toBeTruthy();
   });
+
+  describe('should have a valid name and extension depending of the full filename', () => {
+    it('if the fullname is empty', async () => {
+      const wrapper = await setUp();
+      wrapper.vm.fullName = '';
+      await Vue.nextTick();
+      expect(wrapper.vm.name).toBe('');
+      expect(wrapper.vm.extension).toBe('');
+    });
+
+    it('if the fullname has no dot', async () => {
+      const wrapper = await setUp();
+      wrapper.vm.fullName = 'my_file';
+      await Vue.nextTick();
+      expect(wrapper.vm.name).toBe('my_file');
+      expect(wrapper.vm.extension).toBe('');
+    });
+
+     it('if the fullname has no extension', async () => {
```

Comment:
- indentation
 - `it()` est bien si tu donne un nom de test comme _should work with ..._, dans ton cas, utilise plutôt `test()`, ça rend le code un poil plus lisible